### DISCUSSION TITLE
How to disable no access Github check

### DISCUSSION THREAD

**Discussion by iurage** [vercel/community#1275] [Help] [answered]
### Summary

I am testing out integrating my project on Github with Vercel, but I am constantly seeing this error when a PR is opened by other team members on the project:
`Vercel - Git author must have access to the project on Vercel.`

How do I disable this check for this repo?

**Comment by swarnava** [comment, marked as answer]
Hi there,

This option cannot be disabled as it's necessary to check correct seat and map with the commit. If you have [git configured](https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel) correctly, this error should no longer appear.

**Reply by PhakornKiong** [reply]
I have a monorepo where we have backend and frontend code.

Currently, i want to only add frontend engineer as team member in `vercel`, however, all the backend code is getting `Git author must have access to the project Vercel` error.

How would you advise for such a setup?

**Reply by swarnava** [reply]
You can configure [Ignore build step](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel) to ignore Backend commit to get trigger as preview or production deployment.

**Reply by PhakornKiong** [reply]
> You can configure [Ignore build step](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel) to ignore Backend commit to get trigger as preview or production deployment.

I've already setup the ignore build steps, and it successfully ignore based on the directory.

The issue here is the person commiting is not registered in vercel. And the bot complains in comment as well as in workflow

**Comment by ustun-ed** [comment]
@swarnava we are experiencing the same issue. we don't want to add all backend members of our team (as it would inflate the costs for no reason), only for the builds to be ignored by the "ignore build step".

Could you ask the team to implement an option somewhere where such checks are not registered as failures? Should I create a new question since this is marked as answered?

What is happening right now is that any backend change PR shows 1 failing check, and the backend devs need to see if it is a false alarm from Vercel or a real issue from the other checks.

**Reply by PhakornKiong** [reply]
I share the same sentiment here. Due to this, i was forced to use GitHub actions instead. And the experience was not that great.

**Comment by swarnava** [comment]
There is no way to prevent deployment using Git integration at this moment. For such custom workflow, you need to switch to custom CI / CD based deployment such as GitHub Action, Gitlab Pipeline etc

**Comment by lucagonzalez** [comment]
Any updates on this?
